t/05-scheduler-full.t

```diff
@@ -200,14 +201,9 @@ subtest 're-scheduling and incompletion of jobs when worker rejects jobs or goes
 
 subtest 'Simulation of heavy unstable load' => sub {
     dead_workers($schema);
-    my @duplicated;
 
     # duplicate latest jobs ignoring failures
-    for my $job ($schema->resultset('Jobs')->latest_jobs) {
-        my $duplicate = $job->auto_duplicate;
-        push(@duplicated, $duplicate) if defined $duplicate;
-    }
-
+    my @duplicated = grep { defined } map { $_->auto_duplicate } $schema->resultset('Jobs')->latest_jobs;
```

Comment:
I assume this is equivalent to `grep { defined $_ }`? If that's the case I would prefer the common/explicit form. Or maybe you can get rid of `grep` completely and use `$_->auto_duplicate // ()` within `map`.

Reply:
Well, I wanted to find out what's the "canonical way" for perl but then I remembered the perl motto TIMTOWTDI ;) Ok, will try

Reply:
My motto when using Perl is use the way that sucks the less :-)